frontend/benefit/applicant/src/styles/globals.css

```diff
@@ -0,0 +1,18 @@
+html,
+body {
+  padding: 0;
+  margin: 0;
+}
+
+* {
+  box-sizing: border-box;
+}
+
+ul {
+  list-style: none;
+}
+
+ul, li {
+  margin: 0;
+  padding: 0;
+}
```

Comment:
No newline at the end of the file. We have `.editorconfig` that specifies UNIX style lines to be used (meaning that every line should end with a newline character, even the last one).  Did you use VS Code? You could install some editorconfig plugin, for example https://marketplace.visualstudio.com/items?itemName=EditorConfig.EditorConfig.

Reply:
Weird, I though this was linter stuff, usually is.